src/cephadm/cephadm.py

```diff
@@ -9454,7 +9457,10 @@ def _get_parser():
     parser_bootstrap.add_argument(
         '--ssh-user',
         default='root',
-        help='set user for SSHing to cluster hosts, passwordless sudo will be needed for non-root users')
+        help='set user for SSHing to cluster hosts, sudo will be needed for non-root users')
+    parser_bootstrap.add_argument(
+        '--ssh-password',
+       help='set password for sudoing on cluster hosts if passwordless sudo is not enabled. sshpass is required on each host of cluster')
```

Comment:
I think this line might be what flake8 is complaining about in
```
cephadm.py:9708:8: E131 continuation line unaligned for hanging indent
```
Line numbers don't match up, but it's definitely something with these arg parsing changes